op_builder/npu/no_impl.py

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+# DeepSpeed Team
+
+from .builder import NPUOpBuilder
+
+
+class NPUNotImplementedBuilder(NPUOpBuilder):
```

Comment:
I think both `NPUNotImplementedBuilder` and `CPUNotIMplemetnedBuilder` can have the same name `NotImplementedBuilder`.  The accelerator interface design allows each accelerator backend to implement op builder with same functionality with same name but different implementation.   So `NotImplementedBuilder` have same functionality and could have different implementation in CPU and NPU.   This also opens the door for DeepSpeed to check whether certain OpBuilder is implemented in the future.

Reply:
Sure. Will change to same name next commit. Thanks.